lib/db.c

```diff
@@ -74,12 +74,79 @@ static const uint32_t GRN_TABLE_PAT_KEY_CACHE_SIZE = 1 << 15;
       grn_string_get_normalized(ctx, nstr, &key, &key_size, NULL);\
       block\
       grn_obj_close(ctx, nstr);\
+    } else {\
+      GRN_DEFINE_NAME(table);\
+      ERR(GRN_INVALID_ARGUMENT,\
+          "[key][normalize] failed to normalize: <%.*s> <%.*s>",\
+          (int)name_size, (const char *)name,\
+          (int)key_size, (const char *)key); \
```

Comment:
There is an extra space between `;` and `\`.

Reply:
I've fixed.